site/src/pages/CreateTemplatePage/BuildLogsDrawer.tsx

```diff
@@ -27,6 +31,16 @@ export const BuildLogsDrawer: FC<BuildLogsDrawerProps> = ({
 	variablesSectionRef,
 	...drawerProps
 }) => {
+	const org = templateVersion?.organization_id
+	const {
+		data: compatibleProvisioners,
+		isLoading: provisionerDaemonsLoading,
+		isError: couldntGetProvisioners,
```

Comment:
avoid contractions in variable names
```suggestion
		isError: couldNotGetProvisioners,
```